### PR TITLE
feat: add javascript solution to lc problem: No.0099

### PR DESCRIPTION
**Pull request by ovyas24**

### CODE REVIEW

File: solution/0000-0099/0099.Recover Binary Search Tree/README_EN.md [hunk 1]
```diff
@@ -243,6 +243,51 @@ public class Solution {
 }
 ```
 
+### **Javascript**
```
Comment (YangFong) [inline solution/0000-0099/0099.Recover Binary Search Tree/README_EN.md:246]: JavaScript is recommended

File: solution/0000-0099/0099.Recover Binary Search Tree/README_EN.md [hunk 2]
```diff
@@ -243,6 +243,51 @@ public class Solution {
...
+```javascript
```
Comment (YangFong) [inline solution/0000-0099/0099.Recover Binary Search Tree/README_EN.md:247]: It is recommended to add a line break between the code block and the title

**Comment by YangFong** [comment]
In addition, the solution needs to be added in the Chinese version at the same time, if there is a solution description, you can omit it.

File: solution/0000-0099/0099.Recover Binary Search Tree/README_EN.md (hunk 1 above)
Comment (ovyas24) [reply solution/0000-0099/0099.Recover Binary Search Tree/README_EN.md:246]: Okk will add that

File: solution/0000-0099/0099.Recover Binary Search Tree/README_EN.md (hunk 2 above)
Comment (ovyas24) [reply solution/0000-0099/0099.Recover Binary Search Tree/README_EN.md:247]: Got it

**Comment by YangFong** [comment]
Thank you for your contribution. In addition, I suggest opening a new branch for writing to prevent conflicts.

**Comment by ovyas24** [comment]
Ok will keep that in mind going forward

**Pull request merged**